docs/SupportProgress.md

```diff
@@ -144,7 +144,7 @@ Gluten supports 94 functions. (Draw to right to see all data types)
 | substring_index                |                        |                       |            |                        |         |      |       |      |      |       |        |      |           |        |         |      |        |          |       |      |        |      |
 | startswith                     | startsWith             | S                     |            |                        |         |      |       |      |      |       |        |      | S         |        |         |      |        |          |       |      |        |      |
 | translate                      |                        |                       |            |                        |         |      |       |      |      |       |        |      |           |        |         |      |        |          |       |      |        |      |
-| trim                           | trim                   |                       |            | Mismatched             |         |      |       |      |      |       |        |      | S         |        |         |      |        |          |       |      |        |      |
+| trim                           | trim                   |                       | PS         | Supports only trim(str), not rtrim(BOTH/LEADING/TRAILING \[trimStr\] FROM str) |         |      |       |      |      |       |        |      |         |  S      |         |      |        |          |       |      |        |      |
```

Comment:
Only one small issue:
`not rtrim(` should be changed to `not trim(`, right?

Reply:
done